### PR TITLE
Add endpoints for stats detail

### PR DESCRIPTION
**Pull request by planarvoid**
- This PR adds Store for post detail. The endpoint was already implemented for loading post details in the `LatestPostSummary`. I've only changed the SqlUtils to accept a `postId` of the cached post. It should be tested with the related `WPAndroid` PR

### CODE REVIEW

File: fluxc/src/main/java/org/wordpress/android/fluxc/persistence/InsightsSqlUtils.kt [hunk 1]
```diff
@@ -85,8 +85,8 @@ class InsightsSqlUtils
         return select(site, LATEST_POST_DETAIL_INSIGHTS, PostResponse::class.java)
     }
 
-    fun selectLatestPostStats(site: SiteModel): PostStatsResponse? {
-        return select(site, LATEST_POST_STATS_INSIGHTS, PostStatsResponse::class.java)
+    fun selectLatestPostStats(site: SiteModel, postId: Long): PostStatsResponse? {
```
Comment (0nko) [inline fluxc/src/main/java/org/wordpress/android/fluxc/persistence/InsightsSqlUtils.kt]: We should probably rename this to something like `selectDetailedPostStats()` now that we're storing more than one.

File: fluxc/src/main/java/org/wordpress/android/fluxc/persistence/InsightsSqlUtils.kt [hunk 2]
```diff
@@ -85,8 +85,8 @@ class InsightsSqlUtils
...
+        return select(site, LATEST_POST_STATS_INSIGHTS, PostStatsResponse::class.java, postId)
```
Comment (0nko) [inline fluxc/src/main/java/org/wordpress/android/fluxc/persistence/InsightsSqlUtils.kt]: Same here, I think the enum value should be renamed.

File: fluxc/src/main/java/org/wordpress/android/fluxc/store/stats/insights/LatestPostInsightsStore.kt [hunk 3]
```diff
@@ -43,12 +43,10 @@ class LatestPostInsightsStore @Inject constructor(
     }
 
     fun getLatestPostInsights(site: SiteModel): InsightsLatestPostModel? {
```
Comment (0nko) [inline fluxc/src/main/java/org/wordpress/android/fluxc/store/stats/insights/LatestPostInsightsStore.kt]: Same here: method name.

Comment (planarvoid) [reply fluxc/src/main/java/org/wordpress/android/fluxc/store/stats/insights/LatestPostInsightsStore.kt]: I think this one is correct. This is still returning the stats for the latest post. There is another store that returns the value for a post by `id`

**Comment by planarvoid** [comment]
@0nko  thanks for the review. Everything should be fixed now and there is a WPAndroid PR that can be tested.

**Pull request merged**